flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -2953,17 +2966,20 @@
                     </UseTemplate>
                 </Component>
 
+                <!-- LDG ELEV AUTO -->
                 <Component ID="Overhead_CabinPress">
                     <Component ID="KNOB_OVHD_CABINPRESS_LDGELEV" Node="KNOB_OVHD_CABINPRESS_LDGELEV">
                         <UseTemplate Name="FBW_Continuous_Knob_Limited_From_Simvar">
```

Comment:
Why do we have both `L:A32NX_LANDING_ELEVATION` and `L:XMLVAR_KNOB_OVHD_CABINPRESS_LDGELEV` ?